### PR TITLE
AGW: deploy envoy from magma repo.

### PR DESCRIPTION
**Pull request by pshelar**
Repo URL: https://getenvoy.io/gpg is not reliable, Use
package cached in magma repo.

Signed-off-by: Pravin B Shelar <pbshelar@fb.com>

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

**Review by ardzoht — APPROVED** [review]
lgtm

File: lte/gateway/deploy/magma_dev_focal.yml [hunk 1]
```diff
@@ -48,6 +48,5 @@
         magma_repo: /home/{{ ansible_user }}/magma-packages
         magma_deps: /home/{{ ansible_user }}/magma-deps
     - role: fluent_bit
-    - role: envoy
```
Comment (tmdzk) [inline lte/gateway/deploy/magma_dev_focal.yml:51]: If you remove it from there It won't be install in the devs VM.

**Review by tmdzk — CHANGES_REQUESTED** [review]
Envoy had an issue with their registry but it's fixed now. 

2 solutions: 
- we change the dev role to point to our selfhost version of envoy
- we keep it that way and trust it will never happen again.

**Review by tmdzk — APPROVED** [review]

File: lte/gateway/deploy/magma_dev_focal.yml (hunk 1 above)
Comment (pshelar) [reply lte/gateway/deploy/magma_dev_focal.yml:51]: @tmdzk the magma role would install it from our repo.

**Pull request merged**